plex-playlist-sync/utils/spotify.py

```diff
@@ -96,7 +96,8 @@ def spotify_playlist_sync(
         user_id (str): spotify user id
         plex (PlexServer): A configured PlexServer instance
     """
-    playlists = _get_sp_user_playlists(sp, userInputs.spotify_user_id)
+    playlists = _get_sp_user_playlists(
+        sp, userInputs.spotify_user_id, " - Spotify" if userInputs.append_service_suffix else "")
```

Comment:
Please change this to follow 79-character limit like in rest of the code. I use black formatting btw. Sorry, I haven't added a Contibute.md

Ex:
```
    playlists = _get_sp_user_playlists(sp, userInputs.spotify_user_id)
    playlists = _get_sp_user_playlists(
        sp,
        userInputs.spotify_user_id,
        " - Spotify" if userInputs.append_service_suffix else "",
    )
```